packages/lodestar/src/chain/interface.ts

```diff
@@ -115,5 +115,9 @@ export interface IBeaconChain {
   /**
    * Process a chain of blocks until complete.
    */
-  processChainSegment(signedBlocks: allForks.SignedBeaconBlock[], trusted?: boolean): Promise<void>;
+  processChainSegment(
+    signedBlocks: allForks.SignedBeaconBlock[],
+    prefinalized?: boolean,
+    trusted?: boolean
```

Comment:
Can we use named arguments `{prefinalized?: boolean; trusted?: boolean}` to make it easier to read?